radio/src/tests/mixer.cpp

```diff
@@ -686,33 +686,35 @@ TEST_F(MixerTest, DelayOnSwitch)
 
 TEST_F(MixerTest, DelayOnSwitch2)
 {
-  g_eeGeneral.switchConfig = SWITCH_3POS;
+  // SWSRC_ON is not a valid Switch option for Mixes ????
```

Comment:
SWRSC_ON and SWSRC_ONE are only valid for Special / Global functions (isSwitchAvaileble function in gui_common.cpp).

SWSRC_ON would not work for delays anyway since the delay is only activated on switch state change and SWSRC_ON never changes state. So the DelayOnSwitch2 test will always fail as no delay can be triggered.

Reply:
The easy (and old) use-case I have for understanding how this fits together is that of a servo sweep test function... 

![image](https://github.com/EdgeTX/edgetx/assets/5500713/dba67a38-5bbe-4248-b2ec-83d0b9783f04)
![image](https://github.com/EdgeTX/edgetx/assets/5500713/ed772653-342e-49d8-b658-9430ee44cd0f)

It should sweep for 4s, halt for 0.5s, and then sweep for 4s the other way, halt for 0.5, etc, etc. And is retriggered by the 4.5s LS timer. This is the test I used for the 2.9.2 version of the delay changes, and it worked perfectly there. 

The issue here with the test is that "ON" is not a valid switch input - it cannot be selected in the UI. So the test is faulty in that respect. However, the test previously did pass, so *why* exactly does it now fail?